repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -628,6 +630,15 @@ def _run_ceremony_steps(save: bool) -> Dict[str, Any]:
     tuf_management = TUFManagement(setup, save)
     metadata = tuf_management.initialize_metadata()
 
+    # Inform user for pending signatures
+    vals = metadata.values()
+    root: Metadata[Root] = vals[Roles.ROOT]
+    if root.signatures < setup.threshold[Roles.ROOT]:
+            console.print(
+                "Root is not trustworhy yet. %d pending signatures are left",
+                setup.threshold[Roles.ROOT] - len(root.signatures)
+            )
```

Comment:
It's preferable to use [f-strings](https://peps.python.org/pep-0498/) and you can reuse the calculation you did above
```suggestion
        console.print(
            f"Root is not trustworthy yet. {pending_signatures} pending signatures are left."
        )
```